dns.c

```diff
@@ -47,18 +51,38 @@ int type;
  errno = 0;
  if (!stralloc_copy(&glue,domain)) return DNS_MEM;
  if (!stralloc_0(&glue)) return DNS_MEM;
- responselen = lookup(glue.s,C_IN,type,response.buf,sizeof(response));
+ if (!responsebuflen) {
+  if ((response.buf = malloc(PACKETSZ+1)))
+   responsebuflen = PACKETSZ+1;
+  else return DNS_MEM;
+ }
+
+ responselen = lookup(glue.s,C_IN,type,response.buf,responsebuflen);
+ if ((responselen >= responsebuflen) ||
+     (responselen > 0 && (((HEADER *)response.buf)->tc)))
+  {
```

Comment:
Why is this indented? Should merge with previous line.

Reply:
because the line becomes too long for the ancient 80 char limits or so.

Reply:
No? the line 60 is even longer. I'm talking about the `{` only.

Reply:
This comes from the original patch. Looks like it was trying to match the existing brace style in that file, which at a glance was consistently weird in that way.